### ISSUE TITLE
NVDA gets error on some web-view applications

### ISSUE THREAD

**Issue by shadyar** [nvaccess/nvda#9319]
### Steps to reproduce:
1. after typing in a text box.
2. if you press ESC key.
3. and try to navigate on the page.

### Actual behavior:
NVDA gets error beep.
[nvda.log](https://github.com/nvaccess/nvda/files/2898014/nvda.log)

### Expected behavior:
NVDA should read content of the page 
#### NVDA installed/portable/running from source:
NVDA installed
#### NVDA version:
(NVDA)
Version: alpha-16813,a0b991af
#### Windows version:
windows 10 x64
#### Name and version of other software in use when reproducing the issue:
Babylon
#### Other information about your system:
laptop
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
yes, while using all earlier stable versions of NVDA, this issue did not occur.

**Comment by Adriani90** [comment]
Is this issue also in nvda 2018.4.1?

**Comment by shadyar** [comment]
@Adriani90

**Comment by shadyar** [comment]
@Adriani90  
while using all earlier stable versions of NVDA, this issue did not occur.

**Comment by Adriani90** [comment]
Cc: @mltony, @michaelDCurran

Von meinem iPhone gesendet

**Comment by DrSooom** [comment]
@shadyar: The error sound doesn't occur on RC and stable versions. Please change the log level to "debug" and compare again. And please list the version of Babylon (and all other browsers, where this issue occurs) too. Thanks.

Maybe related: Issue #8967. So please disconnect your braille display too and compare again.

**Comment by shadyar** [comment]
I changed the log level to "debug" and uploaded that log.
yes I have tested The error sound does not occur on RC and stable versions as well as it does not occurred using nvda_snapshot_alpha-16405,19bb21c2 (an older version).
version of Babylon: v9
Babylon gui in all later versions makes a web-view window using likely an internal browser or platform facilities. it is possible navigating   on it by NVDA Browse Mode commands.
I notified there is the following warning in the log.
WARNING - core.main (08:56:08.888):
Java Access Bridge not available
I am sure java access bridge is enabled on my machine because I has enabled it by command line. I think this warning occurred because NVDA dose not find JRE. I am java developer, I have just JDK v8 that is why it is not needed installing JRE (recommended by oracle). JDK has internal JRE. 
@DrSooom 
[nvda.log](https://github.com/nvaccess/nvda/files/2899156/nvda.log)
